config/ftbquests/normal/chapters/6c29cdf5/dd089aeb.snbt

```diff
@@ -19,14 +19,13 @@
 		},
 		id: "simple_trophies:trophy"
 	},
-	x: 16.5d,
-	y: 8.25d,
+	x: 10.0d,
+	y: 6.4d,
 	text: [
 		"{q.botania.the_chapter_is_com.desc}"
 	],
 	hide_dependency_lines: 1b,
 	dependencies: [
```

Comment:
This is a quest for completing chapter. You need to manually add dependencies in game for all the "tail" (no dependants) quests of the chapter. If you wont do that, players will have "Chapter is completed" message before they actually completed all quests.

![](https://i.imgur.com/kVtAKoB.png)